README.md

```diff
@@ -24,4 +24,5 @@ Check out the release notes and [the wiki](https://github.com/WPIRoboticsProject
 
 To build and run, use the included Gradle wrapper script.
 
-    ./gradlew run
+ - With UI `./gradlew :applicationUi:run`
+ - Headless `./gradlew :core:run`
```

Comment:
Does headless mode do anything yet?  If not, we should not include it in the readme